### ISSUE TITLE
Test `lpc::tests::qlpc_recovery` is flaky

### ISSUE THREAD

**Issue by yotarok** [yotarok/flacenc-rs#9]
This test fails at a very low probability and may be indicating potential bugs.

**Comment by yotarok** [comment]
Observation:
```
thread 'lpc::tests::qlpc_recovery::lpc_order_2' panicked at 'assertion failed: `(left == right)`
  left: `0`,
 right: `12`', src/lpc.rs:760:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by yotarok** [comment]
Probably, resolved in [2cca3b594924b153961c92e18d25182911329452](https://github.com/yotarok/flacenc-rs/commit/2cca3b594924b153961c92e18d25182911329452)
